pkg/gatherers/clusterconfig/container_images.go

```diff
@@ -68,6 +69,8 @@ func gatherContainerImages(ctx context.Context, coreClient corev1client.CoreV1In
 		for podIndex, pod := range pods.Items { //nolint:gocritic
 			podPtr := &pods.Items[podIndex]
 			if strings.HasPrefix(pod.Namespace, "openshift-") && check.HasContainerInCrashloop(podPtr) {
+				obfuscateContainerEnvVars(podPtr.Spec.Containers)
```

Comment:
We will need to use this new function on more places - e.g somewhere here https://github.com/openshift/insights-operator/blob/master/pkg/gatherers/clusterconfig/operators_pods_and_events.go#L218